src/nanoemoji/config.py

```diff
@@ -150,15 +154,18 @@ def load(config_file: Path = None, additional_srcs: Tuple[Path] = None) -> FontC
         srcs = set()
         if "srcs" in master_config:
             for src in master_config.pop("srcs"):
-                if Path(src).is_file():
-                    srcs.add(Path(src).resolve())
+                src_path = Path(src)
+                if "*" in src:
+                    srcs |= set(config_dir.glob(src))
```

Comment:
the fact that asterisk is in `src` doesn't necessarily mean the glob should be relative to `config_dir`. I think one could have an absolute path with `*` pattern, no?

Reply:
Good catch, will fix.

Reply:
Done